djangocms_navigation/cms_menus.py

```diff
@@ -11,6 +11,20 @@
 from .utils import get_versionable_for_content
 
 
+class MenuItemNavigationNode(NavigationNode):
+
+    def __init__(self, *args, **kwargs):
+        self.content = kwargs.pop('content')
+        super().__init__(*args, **kwargs)
+
+    def is_selected(self, request):
+        try:
+            content = request.current_page
```

Comment:
current_page in the real world is page. It is now returning false. No test is actually testing the softroot functionality, this prove the fact.

$ type(request.current_page)
Page

$ type(self.content)
PageContent

Reply:
Updated as suggested